cmd/cluster_add.go

```diff
@@ -49,45 +49,43 @@ var addCmd = &cobra.Command{
 		// stores the home dir of the user
 		// used to know the location of .kcm directory as .kcm directory
 		// is in home
+
+		// fmt.Println(args[0])
 		home, err := homedir.Dir()
 		if err != nil {
 			fmt.Println(err)
 			os.Exit(1)
 		}
 
-		if checkFileOrDirectoryExists(home + "/.kcm/" + clusterOptions.clusterName) {
-			fmt.Println("error: clustername should be unique..")
+		if checkFileOrDirectoryExists(home + "/.kcm/" + args[0]) {
```

Comment:
just a thought, instead of using args[0] everywhere should we store it in the variable first and then use that variable (clusterName)?

Reply:
good point